### ISSUE TITLE
.ropeproject cannot be placed outside of source code dir

### ISSUE THREAD

**Issue by codecure** [JulianEberius/SublimeRope#25]
For example, in ropevim i can store my .ropeprojects in ~/RopeProj, and open project from there.
Can SublimeRope do that ?

**Comment by JulianEberius** [comment]
Yes this might be nice, for example when working with projects on some server... I've looked at Rope and the behaviour seems hard-coded there on first sight. When I have some time, I will check how ropevim solved this.

Julian

**Comment by crucialfelix** [comment]
just add a project setting that can point to a .ropeproject

at the moment absolutely nothing works for me, not even an error message.  I guess its because its not finding the .ropeproject which is not in the same dir as the this particular source file
